app/src/main/java/com/example/itbook/data/model/Book.kt

```diff
@@ -13,7 +13,7 @@ class Book(
     val publisher: String,
     val language: String,
     val isbn10: String,
-    val isbn13: String,
+    val isbn13: String,//id to search books
```

Comment:
```suggestion
    val searchId: String
```
Nếu mà id để riêng cho search thì em có thể đặt tên biến này cho Book, hiện tại hình như nó không dùng cho mỗi search thì phải. tương tự vs thằng isbn10

Reply:
dạ cái thằng isbn13 nó hiển thị ở trong information nữa nên em nghĩ là cứ để nguyên, sửa tên biến trong các file kia thôi ạ